src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -2666,6 +2666,7 @@ namespace wallet_rpc
       std::string ssl_ca_file;
       std::vector<std::string> ssl_allowed_fingerprints;
       bool ssl_allow_any_cert;
+      bool is_connected;
```

Comment:
I think that this field would better be named something else like `check_connection` because it's a flag/request to the wallet to do something rather than the response itself.

Reply:
changed to `check_connection` on [e71d7f6](https://github.com/monero-project/monero/pull/8317/commits/e71d7f64509ba39d911b40aa64b71725fcec652f)